examples/scaling-criteo/04-Triton-Inference-with-TF.ipynb

```diff
@@ -374,26 +301,78 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 15,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "POST /v2/repository/index, headers None\n",
+      "get_model_repository_index, metadata ()\n",
       "\n",
-      "<HTTPSocketPoolResponse status=200 headers={'content-type': 'application/json', 'content-length': '62'}>\n",
-      "bytearray(b'[{\"name\":\"criteo\"},{\"name\":\"criteo_nvt\"},{\"name\":\"criteo_tf\"}]')\n"
+      "models {\n",
+      "  name: \"criteo\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"criteo_nvt\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"criteo_tf\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"movielens\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"movielens_nvt\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"movielens_tf\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"rossmann\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"rossmann_nvt\"\n",
+      "}\n",
+      "models {\n",
+      "  name: \"rossmann_tf\"\n",
+      "}\n",
+      "\n"
```

Comment:
I'm not sure if we should be showing all the other models on the server - since this is just for the criteo example

Reply:
yeah I will rerun the outputs of each I guess to get them right